### PR TITLE
Update Azure Docker docs

### PR DESCRIPTION
**Pull request by bmw**
This just updates this comment in response to switching to an access token to authenticate with Docker Hub.

### CODE REVIEW

File: .azure-pipelines/templates/stages/deploy-stage.yml [hunk 1]
```diff
@@ -96,11 +96,16 @@ stages:
               # which was created by following the instructions at
               # https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml#sep-docreg.
               # The name given to this service account must match the value
-              # given to containerRegistry below. "Grant access to all
-              # pipelines" should also be checked.  To revoke these
-              # credentials, we can change the password on the certbotbot
-              # Docker Hub account or remove the account from the
-              # Certbot organization on Docker Hub.
+              # given to containerRegistry below.  The authentication used when
```
Comment (ohemorange) [inline .azure-pipelines/templates/stages/deploy-stage.yml:99]:
```suggestion
              # given to containerRegistry below. The authentication used when
```

File: .azure-pipelines/templates/stages/deploy-stage.yml [hunk 2]
```diff
@@ -96,11 +96,16 @@ stages:
...
+              # creating this service account was a personal access token
+              # rather than a password to bypass 2FA. When Brad set this up,
+              # Azure Pipelines failed to verify the credentials with an error
+              # like "access is forbidden with a JWT issued from a personal
+              # access token", but after saving them without verification, the
+              # access token worked when the pipeline actually ran. "Grant
+              # access to all pipelines" should also be checked on the service
+              # account.  The access token can be deleted on Docker Hub if
```
Comment (ohemorange) [inline .azure-pipelines/templates/stages/deploy-stage.yml:107]:
```suggestion
              # account. The access token can be deleted on Docker Hub if
```

**Review by ohemorange — CHANGES_REQUESTED** [review]
Found a few double spaces, but more importantly, do we think they're going to continue allowing PATs to bypass 2fa? I would have thought so, but the error that didn't seem to matter does give me some pause.

**Comment by bmw** [comment]
> Found a few double spaces

I think the default vim config does this when line wrapping and I've never bothered to change it. Maybe I should.

> do we think they're going to continue allowing PATs to bypass 2fa? I would have thought so, but the error that didn't seem to matter does give me some pause.

I think so! https://docs.docker.com/docker-hub/access-tokens/ says:

> If you have [two-factor authentication (2FA)](https://docs.docker.com/docker-hub/2fa/) enabled on your account, you must create at least one personal access token. Otherwise, you will be unable to log in to your account from the Docker CLI.

I suspect I got the verification error I did because the Azure Pipelines site is trying to verify the PAT using a web interface which Docker Hub doesn't like.

**Review by ohemorange — APPROVED** [review]
That sounds safe enough re: PATs, lgtm!

**Pull request merged**